Source/Core/Core/HW/GCMemcard/GCMemcardBase.h

```diff
@@ -0,0 +1,33 @@
+// Copyright 2020 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include "Common/CommonTypes.h"
+
+#include "Core/HW/GCMemcard/GCMemcard.h"
+
+class PointerWrap;
+
+class MemoryCardBase
+{
+public:
+  explicit MemoryCardBase(int card_index = 0, int size_mbits = Memcard::MBIT_SIZE_MEMORY_CARD_2043)
+      : m_card_index(card_index), m_nintendo_card_id(size_mbits)
+  {
+  }
+  virtual ~MemoryCardBase() {}
```

Comment:
Given this is being migrated, this can be changed to:

```suggestion
  virtual ~MemoryCardBase() = default;
```